sound/soc/sof/intel/apl.c

```diff
@@ -89,6 +89,10 @@ struct snd_sof_dsp_ops sof_apl_ops = {
 	/* firmware run */
 	.run = hda_dsp_cl_boot_firmware,
 
+	/* dsp core enable/disable */
+	.core_enable = hda_dsp_core_run,
```

Comment:
hda_dsp_core_run fails, because core is not powered up.

```suggestion
	.core_enable = hda_dsp_enable_core,
```